src/taphouse.coffee

```diff
@@ -34,9 +34,14 @@ module.exports = (robot) ->
                 ];
 
                 drinks.forEach((drink) ->
-                    response.push(
-                      '(beer) ' + drink.name + ' [' + drink.brewery + '] - ' + drink.formattedPrice
-                    )
+                    output = '(beer) ' + drink.name + ' [' + drink.brewery + '] - '
+
+                    isHalfPint = drink.quantity == 'half'
+
+                    output = output + '½ ' if isHalfPint
```

Comment:
Thanks @rjwaring! 


...I had to write it like this. Originally wrote as a "normal" if statement and it complained. Looked at docs and saw this. It also complained when I used `===`, which isn't valid. It does automatically convert `==` to `===` though, so... (shrug)